samples/queries.js

```diff
@@ -78,7 +78,6 @@ function queryStackOverflow(projectId) {
 // [END bigquery_simple_app_all]
 
 function syncQuery(sqlQuery, projectId) {
-  // [START bigquery_sync_query]
```

Comment:
Don't delete this. This sample is displayed here: https://cloud.google.com/nodejs/

Reply:
Updated the sample include code on devsite to specify a branch tag so the sample render won't break. After this PR is merged, I will update devsite to use bigquery_query instead of bigquery_sync_query.